proxy/envoy/resources.go

```diff
@@ -131,6 +131,8 @@ type FilterMixerConfig struct {
 	// ForwardAttributes specifies the list of attribute keys and values that
 	// are forwarded as an HTTP header to the server side proxy
 	ForwardAttributes map[string]string `json:"forward_attributes,omitempty"`
+
+	QuotaName string `json:"quota_name,omitempty"`
```

Comment:
That requires changes to mixerclient first; lets get this change in to unblock rate limiting and we can circle back to fix up how quotas are actually checked.